e2e/pod.go

```diff
@@ -374,6 +374,7 @@ func deletePod(name, ns string, c kubernetes.Interface, t int) error {
 	})
 }
 
+// nolint:unparam // currently skipNotFound is always false, this can change in the future
```

Comment:
any idea why it's not found in current CI? if it's causing a CI issue it should be a separate commit.

Reply:
no idea , however in yesterday's refresh CI started to complain about it.